src/main/java/me/botsko/prism/Prism.java

```diff
@@ -168,113 +148,132 @@ public static PrismDataSource getPrismDataSource() {
     public static String getPasteKey() {
         return pasteKey;
     }
-
+    /**
+     * Get the plugin name.
+     *
+     * @return String
+     */
+    @SuppressWarnings("WeakerAccess")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['METHOD_DEF' should be separated from previous statement.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)